src/Console/Command/CleanBranchCommand.php

```diff
@@ -59,15 +57,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $yes = $input->getOption('yes') ?: $force;
         $config = Yaml::parse(file_get_contents($input->getOption('config')));
         $pattern = $input->getOption('pattern') ?? $config['CleanBranch']['pattern'];
+        $branchNameFilter = $input->getArgument('branchname-filter');
 
         $finder = new Finder();
         $directories = $finder->depth(0)->directories()->in($folder);
 
         try {
             $issueService = new IssueService(new ArrayConfiguration([
-                'jiraHost'     => $config['CleanBranch']['hostname'],
-                'jiraUser'     => $config['CleanBranch']['username'],
-                'jiraPassword' => $config['CleanBranch']['password']
+                'jiraHost'     => $config['Jira']['hostname'],
```

Comment:
Same here for changed config values

Reply:
I changed the .dist values for convenient usage within tempa.
What do you want me to change here exactly?

Reply:
Simply the keys itself like for mysql => `mysql_hostname`
So we don't have `hostname` to be used with `tempa` but instead `jira_hostname`